src/liquid/components/ld-pagination/ld-pagination.tsx

```diff
@@ -71,6 +74,15 @@ export class LdPagination {
   /** Number of items permanently visible at the start/end. */
   @Prop() sticky = 0
 
+  /** Items display mode, default as numbers */
+  @Prop() mode?: 'numbers' | 'dots' = 'numbers'
+
+  /** Switch colors for brand background */
+  @Prop() brandColor?: boolean
```

Comment:
```suggestion
  /** Items display mode, default as numbers. */
  @Prop() mode?: 'numbers' | 'dots' = 'numbers'

  /** Switch colors for brand background. */
  @Prop() brandColor?: boolean
```

Just for consistency.